src/utils/sizingText.js

```diff
@@ -0,0 +1,6 @@
+export const sizingText = (calculating, result) => {
```

Comment:
```suggestion
export const getTextSize= (calculating) => {
  if (calculating.length > 15 && calculating.length <= 38)
   return "font-size: 15px";
  return "font-size: 35px";
};
```
и потом присваеваем эти стили в месте, где ты вызываешь эту функцию